style.css

```diff
@@ -430,3 +439,75 @@ a:hover {
   flex-direction: column;
   background-color: #1c1a19;
 }
+
+.about-section hr {
+  width: 40vh;
+  color: #fff;
+  margin-top: 220px;
+}
+
+.form-section {
+  margin: 120px 24px;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.form-desc {
+  font-family: Inter, "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  text-align: center;
+  color: #172b4d;
+  margin: 24px;
+  width: 327px;
+  height: 216px;
+}
+
+.form-section form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 70px 0;
+  width: 100%;
+}
+
+input {
+  width: 90%;
+  height: 20px;
+  border-top-style: none;
+  border-inline-style: none;
+  padding: 12px;
+  font-size: 15px;
+  margin-top: 30px;
+  color: #979493;
+  font-family: Inter, sans-serif;
+}
+
+textarea {
+  padding: 7px;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+  resize: vertical;
+  background-color: #fbf8f7;
+  margin-top: 30px;
+  border: none;
+  width: 327px;
```

Comment:
Let's try and make the width 100% so it can take the width of the form. the textarea width is small compared to the actual design.
Your design
![me](https://user-images.githubusercontent.com/53356820/144310914-1b6f6f3f-7c53-4edc-87bb-f5d0f855c983.png)
Figma design
![me1](https://user-images.githubusercontent.com/53356820/144310960-7d7c0cf7-3d78-47b8-93f3-14dfb687ba9d.png)

Reply:
Hi Keneogu, Thanks for your Review but this should be an optional suggestion and not a required one due to all main activities where completed

Reply:
```suggestion
  width: 100%;
```

Changes implemented